### DISCUSSION TITLE
Visuelle retningslinjer for mønster og maler

### DISCUSSION THREAD

**Discussion by Lilyngs** [navikt/Designsystemet#3088] [Mønster og maler]
Visuelle retningslinjer for brev er klar til bruk. Når du har fått prøvd ut og høstet erfaringer ønsker vi å lære av din innsikt. Delta i diskusjonen og bli med på å forbedre retningslinjene.

**Comment by B101147** [comment]
Kjempeflott at dette nå er tilgjengelig på Aksel.   

Brev produseres i dag for tilgjengeliggjøring på ulike plattformer, inkludert papir, så en ting det kan være lurt å legge inn er felles forståelse av avstander og størrelser.  Dette gjelder pixler, mm og points, slik at det er lik forståelse og slik at man unngår ulike omregninger.

**Reply by Lilyngs** [reply]
Takk for tilbakemelding! Dette er i første omgang basert på brevarbeid for produktteam i Pensjon. Siden vi jobber digitalt ble utganspunktet dette. Vi ser behovet for de andre måleenhetene, og håper å få sett på det etterhvert.

**Reply by B101147** [reply]
Hei @Lilyngs , takk for tilbakemeldingen.   Det er veldig mye bra med innholdet, men som nevnt under er det noe som ikke kan benyttes på tvers av ulike løsninger.  Dette er samtidig noe som har vært etterspurt lenge fra CCM ettersom teamet har brev- og brevmaler i NAV som samlet utgjør over 80 prosent av brevproduksjon fra NAV.   
Teamet består primært av brevmal utviklere med lang erfaring i å designe brev og maler i ulike løsninger.   Dersom teamet kan bistå med noe for å gjøre sidene mer nyttig på tvers av løsninger, så vil dette også være enklere for andre å implementere designkravene.

**Reply by Lilyngs** [reply]
Hei og takk for svar. Det er veldig nyttig for oss å vite om. Takk for at dere informerer og gir tilbakemelding. Vi vil gjerne komme tilbake til dere så snart vi kan.

**Comment by maxguld** [comment]
Størrelser og avstander oppgitt som antall piksler/px i "Visuelle retningslinjer for brev" kan ikke brukes i PDF-er for utskrift. Piksler (px) er for skjermvisning og avhengig av oppløsning. Vi trenger skriftstørrelse oppgitt i antall punkter i en gitt skrifttype, og avstander oppgitt i mm eller cm.
Et absolutt minimum må være å oppgi hvilken oppløsning det er snakk om for avstander, slik at vi kan prøve å regne om til de måleenhetene vi vanligvis bruker. Dette gjelder da kun for avstander, mens skrifttyper må vi få som beskrevet i forrige avsnitt.

**Reply by Lilyngs** [reply]
Takk for tilbakemelding! Dette er i første omgang basert på brevarbeid for produktteam i Pensjon. Siden vi jobber digitalt ble utganspunktet dette. Vi ser behovet for de andre måleenhetene, og håper å få sett på det etterhvert.

**Reply by maxguld** [reply]
Team CCM produserer de fleste av NAVs brev, også Pensjon sine. Det hadde vært en stor fordel for NAV om det igjen lages felles retningslinjer for etaten, og at Pensjon ikke lager sitt eget brevoppsett.
Hele poenget skal jo være å gi en felles visuell profil for etaten.

**Comment by maxguld** [comment]
Det er eksempel på brevutforming, men det mangler eksempel på layout med **mottakeradresse** og **returadresse** som skal vises i konvoluttvindu. Adresseområdet må oppfylle Postens krav.

**Reply by Lilyngs** [reply]
Takk for tilbakemelding. Det er nyttig du melder fra. Ja, i denne omgang bare kunnet ta høyde for brev uten mottakeradresse og returadresse. De visuelle retningslinjene er basert på brevarbeid til team Pensjonsbrev.
